### ISSUE TITLE
Update to latest version

### ISSUE THREAD

**Issue by nlohmann** [Klaim/nlohmann-json#3]
This package uses version 3.11.2. The most recent release is 3.11.3.

**Comment by Klaim** [comment]
Thanks for notifying! I'll look for help to publish that version and also fix #1  as I lack time these days to maintain it myself. Worse case it'll be fixed in a few months when I can get back to this.

**Comment by Klaim** [comment]
I'll do a basic version release for now and see later for the other improvements.

**Comment by Klaim** [comment]
Both `v3.11.2` and `v3.11.3` were not published, now they are queued for publishing
- https://queue.cppget.org/nlohmann-json/3.11.2
- https://queue.cppget.org/nlohmann-json/3.11.3

**Comment by nlohmann** [comment]
I think the issue can be closed, right?

**Comment by Klaim** [comment]
Yes I was waiting for the publishing to be done but I forgot to close this.
